src/OpenTelemetry.AutoInstrumentation/Logger/LogBuilderExtensions.cs

```diff
@@ -52,11 +52,14 @@ public static ILoggingBuilder AddOpenTelemetryLogs(this ILoggingBuilder builder)
 
                 options.IncludeFormattedMessage = settings.IncludeFormattedMessage;
 
-                pluginManager.ConfigureLogsOptions(options);
+                pluginManager?.ConfigureLogsOptions(options);
```

Comment:
Any ways if the `PluginManager` can be actually `null` here?

Reply:
Only if some crazy scenarios without calling `OpenTelemetry.AutoInstrumentation.Instrumentation.Initialize` or some crazy race condition.

I do not see anything abviout, but it should be safe.